src/components/Camera/CameraTypes.js

```diff
@@ -25,6 +25,7 @@ type CameraCommonType = {
   cameraError: Object,
   cameraErrorFallback?: string => React.Node,
   cameraErrorHasBackdrop?: boolean,
+  cameraErrorIsDismissible?: boolean,
```

Comment:
Do we need this? Can we not just use `cameraError.type === 'warning'` to determine if the error is dismissible?

Reply:
Yes, I think it definitely sounds like a better approach.

@seewah? Should all warnings become dismissible?

Reply:
i would say so, but I am not sure what are all the warnings. It would be good if i can go over all the screens, and check. Is there a way for me to do that easier, via inspect element?

Reply:
Thanks @seewah.

@stefaniacardenas , that means we do need a specific prop to define dismissible errors

Reply:
Ok, makes sense.